homeassistant/components/binary_sensor/__init__.py

```diff
@@ -147,21 +149,18 @@ async def async_unload_entry(hass, entry):
 class BinarySensorEntity(Entity):
     """Represent a binary sensor."""
 
+    _attr_is_on: bool | None = None
+
     @property
-    def is_on(self):
+    def is_on(self) -> bool | None:
         """Return true if the binary sensor is on."""
         return None
```

Comment:
`_attr_is_on` is not used.

Reply:
Well spotted, opening a fix for that 👍 (after CI has been fixed)